dd-trace-ot/src/main/java/datadog/opentracing/DDTracer.java

```diff
@@ -197,6 +198,14 @@ private DDTracer(
       final LogHandler logHandler,
       final StatsDClient statsDClient) {
 
+    // Check if the tracer is already installed by the agent
+    // Unable to use "instanceof" because of class renaming
+    if (GlobalTracer.get().getClass().getName().equals("datadog.trace.agent.core.CoreTracer")) {
+      log.warn(
```

Comment:
Since we're throwing an exception, perhaps this should be

```suggestion
      log.error(
```